packages/chains/src/arbitrum/redemptionStrategies.ts

```diff
@@ -1,7 +1,11 @@
 import { assetSymbols, RedemptionStrategyContract, underlying } from "@midas-capital/types";
 
-import { assets } from "./assets";
+import { assets, USDC } from "./assets";
 
-const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, string] } = {};
+const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, string] } = {
+  [underlying(assets, assetSymbols.saddleFraxBP)]: [RedemptionStrategyContract.SaddleLpTokenLiquidator, USDC],
+  [underlying(assets, assetSymbols.saddleFraxUsdsBP)]: [RedemptionStrategyContract.SaddleLpTokenLiquidator, USDC],
+  [underlying(assets, assetSymbols.saddleFraxUsdtBP)]: [RedemptionStrategyContract.SaddleLpTokenLiquidator, USDC],
```

Comment:
use USDs as the output token for these two LP tokens and then add another line that converts USDs to USDC